app/assets/stylesheets/spree/backend/solidus_paypal_braintree.scss

```diff
@@ -1,28 +1,4 @@
 /*
-Placeholder manifest file.
-the installer will append this file to the app vendored assets here: 'vendor/assets/stylesheets/spree/backend/all.css'
+ * This CSS file is deprecated. Require spree/backend/solidus_braintree instead.
```

Comment:
I don't know if we have a convention or tool for deprecating CSS files.

Reply:
We could explore using https://sass-lang.com/documentation/at-rules/warn somehow when that file is loaded.

Reply:
Thanks @kennyadsl I'll try that!

Reply:
I updated the install generator to rename the asset files, so there is no more need for the deprecated asset files :+1: